admin_guide/managing_projects.adoc

```diff
@@ -0,0 +1,178 @@
+binding] will deny permissions for self-provisioning any new projects. When
+disabling self-provisioning, set the `projectRequestMessage` parameter in the
+*_master-config.yaml_* file instructing developers on how to request a new
+project. This parameter is a string that will be presented to the developer in
+the web console and command line when they attempt to self-provision a project.
+For example:
+
+----
+Contact your system administrator at projectname@example.com to request a project.
+----
+
+or:
+
+----
+To request a new project, fill out the project request form located at
+https://internal.example.com/openshift-project-request.
+----
+
+[[using-node-selectors]]
+== Using Node Selectors
+
+Node selectors are used in conjunction with labeled nodes to control pod placement. 
+
+[NOTE]
+====
+Labels can be assigned link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+=== Setting the Cluster-wide Default Node Selector
+
+As a cluster administrator, you can set the cluster-wide default node selector to restrict pod placement to specific nodes. 
+
+Edit the *_/etc/origin/master/master-config.yaml_* master configuration file and add a node selector value. This applies the node selector to pods created in all projects without a specified `*nodeSelector*` value:
+
+====
+----
+...
+projectConfig:
+  nodeSelector: "type=user-node,region=east"
+...
+----
+====
+
+Ensure to restart the master OpenShift service for the changes to take effect:
+
+====
+----
+# systemctl restart atomic-openshift-master
+----
+====
+
+=== Setting the Project-wide Node Selector
+
+To create an individual project with a node selector, use the `--node-selector` option when creating a project. For example, if you have an OpenShift topology with multiple regions, you can use a node selector to restrict specific OpenShift projects to only deploy pods onto nodes in a specific region. 
+
+The following creates a new project named `myproject` and dictates that pods be deployed onto nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project myproject \
+    --node-selector='type=user-node,region=east'
+----
+====
+
+Once this command is run, this becomes the adminstrator-set node selector for all pods contained in the indicated project.
+
+[NOTE]
+====
+While the `new-project` subcommand is available for both `oadm` and `oc`, the cluster administrator and developer commands respectively, creating a new project with a node selector is only available with the `oadm` command. The `new-project` subcommand is not available to project developers when self-provisioning projects.
+====
+
+Using the `oadm new-project` command adds an `annotation` section to the project configuration. Changing the `openshift.io/node-selector` value overrides the default:
```

Comment:
On Wed, Mar 2, 2016 at 12:20 AM, Timothy Poitras <notifications@github.com>
wrote:

> In admin_guide/managing_projects.adoc
> <https://github.com/openshift/openshift-docs/pull/1490#discussion_r54680881>
> :
>
> > +Using the `oadm new-project` command adds an `annotation` section to the project configuration. Changing the `openshift.io/node-selector` <http://openshift.io/node-selector> value overrides the default:
>
> Should you mention the location of the project configuration file?
>
>

At this point we're not talking about the project configuration file. We're
talking about the object that represents the project in the API, so that
users have some idea where they can edit it after creation.